install/scripts/install-hipsycl.sh

```diff
@@ -4,14 +4,14 @@ HIPSYCL_PKG_LLVM_VERSION_MINOR=${HIPSYCL_PKG_LLVM_VERSION_MINOR:-0}
 HIPSYCL_PKG_LLVM_VERSION_PATCH=${HIPSYCL_PKG_LLVM_VERSION_PATCH:-1}
 HIPSYCL_PKG_LLVM_REPO_BRANCH=${HIPSYCL_PKG_LLVM_REPO_BRANCH:-release/${HIPSYCL_PKG_LLVM_VERSION_MAJOR}.x}
 
-export HIPSYCL_INSTALL_PREFIX=${HIPSYCL_INSTALL_PREFIX:-/opt/hipSYCL}
+export HIPSYCL_INSTALL_PREFIX=${HIPSYCL_INSTALL_PREFIX:-/home/sbalint/hipSYCL-installation}
```

Comment:
Can we set the default maybe to something like `$HOME/hipSYCL` so that it also works for other users out of the box? We occasionally have users that use these scripts :-)

Reply:
I didn't intend to leave this in here sorry, I will remove it